dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/data/Graph.kt

```diff
@@ -116,7 +135,7 @@ data class Graph(
             PeriodType.SixMonthlyApril,
             PeriodType.SixMonthlyNov,
             -> {
-                val date = minDate().plusMonths(numberOfSteps)
+                val date = minDate()?.plusMonths(numberOfSteps)
```

Comment:
need to be minDate nullable? could this lead to a null pointer in the next line?

Reply:
No as the method returns the local date if anything is null, although I understand it could lead to confusion so I have updated the method to better reflect the return value, it is not nullable anymore